python/air/dialects/_air_ops_ext.py

```diff
@@ -227,6 +229,18 @@ def __init__(
 dma_memcpy_nd = DmaMemcpyNd
 
 
+def air_module(module_function):
```

Comment:
I think the name `air_module` may be misleading as the ModuleOp comes from the `BuiltIn` MLIR dialect. You can just name it `module`.

Reply:
Ah the builtin dialect already has a `module` python binding as well.

Reply:
I'm very open to other suggestions, but I think I'll try to rename it ```module_builder``` unless there are more suggestions!

Reply:
@AndraBisca Thanks also for pointing out the ```module``` decorator! I updated the PR description to reflect my experiences with it.

TL;DR - it works, but doesn't seem very clean so I think it's still worth defining a new decorator. If you have any further thoughts (or if you suspect I am using ```@module``` incorrectly), please let me know!